src/identity/core/getName.ts

```diff
@@ -1,15 +1,58 @@
-import { publicClient } from '../../network/client';
+import { mainnet } from 'viem/chains';
 import type { GetName, GetNameReturnType } from '../types';
+import L2ResolverAbi from '../../abis/L2ResolverAbi';
```

Comment:
I think, is best we keep the ABI inside the identity module.

So just do `src/identity/abi/L2Resolver`, and the reason we keep the wording single, it's just a best practice in folder naming.